apps/demo/project.json

```diff
@@ -41,8 +41,8 @@
           "budgets": [
             {
               "type": "initial",
-              "maximumWarning": "500kb",
```

Comment:
Can we keep the budget with the default values? Can we keep below the threshold by layz loading routes? I think we should try to fix our code before touching the default settings?

Reply:
Lazy loading the routes worked using the default settings

Reply:
Great!